example/serializers.py

```diff
@@ -37,7 +37,7 @@ class Meta:
 
 class BlogSerializer(serializers.ModelSerializer):
     copyright = serializers.SerializerMethodField()
-    tags = TaggedItemSerializer(many=True, read_only=True)
+    tags = TaggedItemDRFSerializer(many=True, read_only=True)
```

Comment:
why has this changed? I guess we should stick to the DJA version here as previous.

Reply:
Oh I thought it was under BlogDRFSerializer. Was careless.